src/app/article-list/article-list.component.css

```diff
@@ -0,0 +1,14 @@
+.articles-list-section{
+    border: solid black 2px;
```

Comment:
`solid black 2px` should be in this order `2px solid black`. `border-width -> border-style-> border-color`

- Reference:  https://www.w3schools.com/css/css_border.asp

Reply:
**Changes Implemented**